src/resources/controllers/route-resources-enabled-subscriber/index.ts

```diff
@@ -0,0 +1,17 @@
+import { createSubscriber } from 'react-sweet-state';
+
+import {
+  RouterStore,
+  type EntireRouterState,
+  type AllRouterActions,
+} from '../../../index';
+import type { RouteWithResources } from '../../common/types';
+
+export const RouteResourceEnabledSubscriber = createSubscriber<
```

Comment:
This is fine for now, but if these were separate packages then you'd want to use `createRouterSelector` as that is part of the public API

Reply:
Something like this?
```
export const RouteResourceEnabledSubscriber = createRouterSelector(state =>
  Boolean(state.route && (state.route as RouteWithResources).resources)
);
```

Reply:
I would be happy to kill it, personally. It would look more like this though, which you could provide in the patch notes

```javascript
const useHasRouteResources = createRouterSelector(state =>
  Boolean(state.route && (state.route as RouteWithResources).resources)
);

export const RouteResourceEnabledSubscriber = ({ children }) => {
  const hasResources = useHasRouteResources();

  return children(hasResources);
};
```

Reply:
Let's kill it